### DISCUSSION TITLE
Discussions!

### DISCUSSION THREAD

**Discussion by nonamecrackers2** [nonamecrackers2/crackers-wither-storm-mod#1357] [Announcements]
Since a lot of people here don't have access to the [Discord server](https://discord.com/invite/cracker-s-modded-community-987817685293355028) (which you should totally join by the way), I thought I should open this up so people can ask questions related to the mod.

We prefer to have general discussions in the Discord server since this place will be less moderated, but if you have general questions about updates, release dates, etc. feel free to ask them here.

**Comment by myron-obsolete-account** [comment]
I don't know what to put here but I found a weird spawn
![2023-08-31_13 31 06](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/2b51b883-3e82-49d5-9edb-0d5f52f99dd8)

**Reply by BluePineappleGuy** [reply]
this has happened to me as well, like bruh what the illagers doin now

**Reply by myron-obsolete-account** [reply]
they are summoning the Wither Storm for the player now >:)

**Reply by nonamecrackers2** [reply]
That’s interesting lol

**Reply by SmackyCats** [reply]
"Illavor, no!"

**Comment by myron-obsolete-account** [comment]
wait I have an idea how about some kind of wither storm screenshot contest and the winner gets something idk what though lmao

**Comment by BluePineappleGuy** [comment]
Made a resource pack that changes the F-Bomb theme so thought i'd post it here for everyone
[Better Formidibomb Theme.zip](https://github.com/nonamecrackers2/crackers-wither-storm-mod/files/12591608/Better.Formidibomb.Theme.zip)

**Comment by myron-obsolete-account** [comment]
I prefer not to use discord to post it there but I made artworks a while back, heavily based on this mod, and uhh idk yeah
![ws1](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/c78e08db-45f1-4a8d-bb34-92a07900e630)
![ws2](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/3a1d95cd-2dee-4791-a0a8-3d1b18a4dd7e)
![ws3](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/6c2511a6-2021-4428-abc5-7c26cfbd3685)
![ws4](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/f4ccd97f-c491-4980-a884-282ec8b4af68)
![ws5](https://github.com/nonamecrackers2/crackers-wither-storm-mod/assets/53058491/bf8a5ad8-1b85-4eb5-b2fa-1d6f87e9cd56)

**Reply by BluePineappleGuy** [reply]
this makes the idea of the wither storm being real 10 times more scary.
should be a movie

**Reply by myron-obsolete-account** [reply]
Bro I literally have the idea of a wither storm movie in my head for like 2 years now but I am not even good in blender so it'll never be real unfortunately. If i could ever make a movie I'd make it heavily based off of cracker's wither storm mod, epic F-Bomb scene etc. and it should be very realistic looking cus my idea was to have like, real people in it, or at least real looking animated people

**Reply by BluePineappleGuy** [reply]
if you ever do make one, share it here

**Reply by Cod1ngGamer** [reply]
I'm trying to learn blender just so I can make this lol

**Comment by nonamecrackers2** [comment]
Those are really well done! Nice work

**Reply by myron-obsolete-account** [reply]
thank you

**Reply by myron-obsolete-account** [reply]
tf am I doing I am supposed to thank you for creating this mod in the first place

**Comment by NinjaSam5399** [comment]
I have Discord. :trollface:

**Comment by BluePineappleGuy** [comment]
gonna comment here so people get notified and remember this place exists so they don't get off topic on issues

(i am guilty of this myself)

**Comment by Cod1ngGamer** [comment]
This may seem a little weird, but does the WS spawn house have a different design depending on the biome it's in? I think I saw an acacia version in an acacia biome once.

**Reply by BluePineappleGuy** [reply]
It does have different variants, this was added in Evolution pt. 1

**Reply by Cod1ngGamer** [reply]
How do I get the order temple spawn platform to spawn in naturally?

**Reply by nonamecrackers2** [reply]
It has a more rare chance to spawn in forest/plains biomes I believe

**Comment by Cod1ngGamer** [comment]
The 2.2 changelog and 3.2 changelog in the wiki are exactly the same, I assume this is a bug.

**Reply by BluePineappleGuy** [reply]
im pretty sure they were the same update, idk why they are like that

**Comment by ahmetenesay** [comment]
Bro, put an alarm on my grave in case the latest version of the mod comes out.
